react-renderer/src/Arrow.tsx

```diff
@@ -23,13 +23,18 @@ class Arrow extends React.Component<IGPIPropsDraggable> {
     const arrowHeadId = "arrowhead_" + shape.name.contents;
     const strokeDasharray = style === "dashed" ? "7, 5" : "";
 
-    // HACK: fix arrow position
+    // HACK: scale path down a bit to accommodate arrow length
     const { width, height } = arrowheads[arrowheadStyle];
     const slope = Math.atan2(ey - sy, ex - sx);
-    const [offsetX, offsetY] = [
-      (Math.cos(slope) * width) / 2,
-      (Math.sin(slope) * height) / 2
-    ];
+      const [offsetX, offsetY] = [0.0, 0.0];
```

Comment:
This is causing the CI to fail. If the current solution doesn't work perfectly, consider reverting this change. We can look into more accurate arrow placement together soon.